src/search/FindBar.js

```diff
@@ -281,6 +285,17 @@ define(function (require, exports, module) {
                     self.trigger("doFind");
                 }
             })
+            .on("click", ".dropdown-icon", function (e) {
+                if (!self.searchField) {
+                    self.showSearchHints();
+                } else if ($(".quick-search-container").is(':visible')) {
+                    $(".quick-search-container").hide();
+                } else if (!$(".quick-search-container").is(':visible')) {
+                    self.searchField.setText($("#find-what").val());
+                    $(".quick-search-container").show();
+                }
+                self.$("#find-what").focus();
```

Comment:
Actually there are chances of having multiple find bars present (one is closing, other might be opening), so there can be multiple `$("#find-what")`, so to add context to it we use `self.$("#find-what")`